stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/MediaAttachmentView.kt

```diff
@@ -71,6 +71,14 @@ internal class MediaAttachmentView : ConstraintLayout {
             }
         }
 
+        if (attachment.type == ModelType.attach_giphy) {
+            val params = binding.imageView.layoutParams
+
+            binding.imageView.layoutParams = params.apply {
+                height = 200.dpToPx()
```

Comment:
What do you think about making it customizable? Does it make sense?

Reply:
I think that making it customizable makes total sense. But I think it should be only height so we don't conflict with the `MaxPossibleWidthDecorator` all messages are limited in width by this decorator and it would be confusing to make only this view as an exception.

Reply:
About the fixed height of all the attachments. The pictures don't cause a glitch in our scroll because we don't need to load then using coil (or any image loader), so I would say that allow then to change its height accordingly with the picture is fine. What do you think?

Reply:
I think we can keep the old logic for other attachments for now